src/main/resources/landing/translation.json

```diff
@@ -2,14 +2,14 @@
     "translations": {
         "en": {
             "serviceTitle": "Appeal to the First-tier Tribunal",
-            "banner": "This is a new service – your <a  class='govuk-link' href='#'>feedback</a> will help us to improve it.",
+            "banner": "<b> Help us improve this service</b><br>This is a new service.Help us for others to improve it by <a  class='govuk-link' href='https://www.smartsurvey.co.uk/s/Specials_Feedback/?pageurl=Specials' target=_blank>giving your feedback</a>",
```

Comment:
The pageurl of the landing page should probably be empty (assuming it's the relative url they want).

Reply:
If i leave it empty then i believe the in-page feed back will not work.